src/packages/sdk/package.json

```diff
@@ -39,7 +39,7 @@
     "prettier": "^2.0.4",
     "ts-jest": "^25.3.1",
     "ts-node": "^8.8.2",
-    "typescript": "^3.8.3"
+    "typescript": "^4.0.0-dev.20200512"
```

Comment:
They released 3.9.2 just 13 hours ago :P Not sure why you need the 4 dev?

Reply:
Yep, tried the dev one out because of a bug, but I can go back to 3.8.3

Reply:
It was btw this bad boy that I was debugging https://github.com/microsoft/TypeScript/issues/38532

Reply:
Ok Thanks! :)